src/spell/modules/spell/spell/src/script.rs

```diff
@@ -77,10 +77,11 @@ pub fn script_cid() -> CIDv1Value {
 #[test_env_helpers::after_each]
 #[cfg(test)]
 mod tests {
+    use marine_rs_sdk::CallParameters;
+    use marine_rs_sdk::ParticleParameters;
     use marine_rs_sdk_test::marine_test;
-    use marine_rs_sdk_test::CallParameters;
 
-    use crate::schema::DB_FILE;
+    const DB_FILE: &str = "./tests_artifacts/spell.sqlite";
```

Comment:
why don't import it? how does it work now?

Reply:
[DB_file](crate::schema::DB_FILE;) it is file inside the wasm runtime, DB_FILE he it is a file in the current system